Cargo.toml

```diff
@@ -14,10 +14,16 @@ cgmath = { version = "0.17", features = ["mint"] }
 miniquad_text_rusttype = { git = "https://github.com/not-fl3/miniquad_text_rusttype" }
 miniquad = "=0.2.55"
```

Comment:
miniquad 0.2 is seriously outdated, there are lots, lots of bugfixes in `0.3.0-alpha.37`

Reply:
ok, I'll try to bump it to the latest version of miniquad then ;)